specifications/api/swagger.yaml

```diff
@@ -281,6 +281,22 @@ x-walletAccountPubkey: &walletAccountPubkey
   maxLength: 128
   example: 1423856bc91c49e928f6f30f4e8d665d53eb4ab6028bd0ac971809d514c92db11423856bc91c49e928f6f30f4e8d665d53eb4ab6028bd0ac971809d514c92db1
 
+x-cosigner: &cosigner
+  type: string
+  pattern: "^(cosigner#)[0-9]*$"
+
+x-cosigners: &cosigners
+  description: Map of cosigners and their account public keys. Use key as in &cosigner, eg. "cosigner#"
+  type: object
+  additionalProperties: *walletAccountPubkey
+  # Note: propertyNames pattern not supported in current OpenAPI version.
+  # propertyNames:
+  #   pattern: '^(cosigner#)[0-9]*$'
+  example:
+    0: { "cosigner#0": "1423856bc91c49e928f6f30f4e8d665d53eb4ab6028bd0ac971809d514c92db11423856bc91c49e928f6f30f4e8d665d53eb4ab6028bd0ac971809d514c92db1" }
+    1: { "cosigner#0": "1423856bc91c49e928f6f30f4e8d665d53eb4ab6028bd0ac971809d514c92db11423856bc91c49e928f6f30f4e8d665d53eb4ab6028bd0ac971809d514c92db1"
+       , "cosigner#3": "1423856bc91c49e928f6f30f4e8d665d53eb4ab6028bd0ac971809d514c92db11423856bc91c49e928f6f30f4e8d665d53eb4ab6028bd0ac971809d514c92db2" }
```

Comment:
good idea to add an example :+1: 
Don't you think however that showing multiple examples may be a bit confusing for people not really familiar with OpenAPI? Shouldn't this stick to let's say, the second example from the key `1`; I think most users would treat the example as if it was a plausible JSON they could use for input; so in this case, an object where keys are cosigners indexes and values are extended public keys.

Reply:
>I think most users would treat the example as if it was a plausible JSON they could use for input

:facepalm: that's what I did!
see **Examples in API doc** -> https://github.com/input-output-hk/cardano-wallet/pull/2549#issuecomment-811186619

Reply:
Yes... especially since Redoc seems to "be smart" and render examples of parent objects using example of child objects.. leading to even more confusion -- if not simply an invalid example altogether.

Reply:
done